Telie/Cells/ReactionsReactionTableViewCell.swift

```diff
@@ -0,0 +1,133 @@
+        timeSinceLabel.font = .systemFont(ofSize: 12, weight: .light)
+        timeSinceLabel.backgroundColor = .clear
+        contentView.addSubview(timeSinceLabel)
+        
+        reactionContentLabel.textColor = .black
+        reactionContentLabel.textAlignment = .left
+        reactionContentLabel.font = .systemFont(ofSize: 16, weight: .medium)
+        reactionContentLabel.backgroundColor = .white
+        reactionContentLabel.layer.cornerRadius = 8
+        reactionContentLabel.layer.masksToBounds = true
+        reactionContentLabel.layer.borderWidth = 1
+        reactionContentLabel.layer.borderColor = UIColor.lightGray2.cgColor
+        
+   
+        reactionContentLabel.paddingLeft = 15
+        reactionContentLabel.paddingRight = 15
+        reactionContentLabel.paddingTop = 10
+        reactionContentLabel.paddingBottom = 12
+        reactionContentLabel.numberOfLines = 0
+
+        contentView.addSubview(reactionContentLabel)
+
+
+        setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        profileImageView.kf.setImage(with: URL(string: Constants.User.defaultImage))
+    }
+    
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+        reactionNameLabel.backgroundColor = selected ?  .lightGray2 : .clear
+    }
+
+    func configure(reactionName: String, reactionProfilePic: String, reactionContent: String, timeSince: String) {
+        reactionNameLabel.text = "    \(reactionName)"
+        reactionContentLabel.text = reactionContent
+        
+       let imageUrl = URL(string: reactionProfilePic)
+        profileImageView.kf.setImage(with: imageUrl)
+        timeSinceLabel.text = timeSince
+    }
+    
+
+
+    private func setupConstraints() {
+        let verticalPadding: CGFloat = 11
+        
+        profileImageView.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(20)
+            make.size.equalTo(CGSize(width: 40, height: 40))
+        }
+        
+        reactionNameLabel.snp.makeConstraints { make in
+            make.leading.equalTo(profileImageView).inset(30)
+            make.trailing.equalToSuperview().inset(20)
+            make.centerY.equalTo(profileImageView)
+            make.height.equalTo(35)
+        }
+        
+        timeSinceLabel.snp.makeConstraints { make in
+            make.trailing.equalToSuperview().inset(20)
+            make.height.equalTo(12)
+            make.centerY.equalTo(reactionNameLabel)
+        }
+        
+        reactionContentLabel.snp.makeConstraints { make in
+            make.top.equalTo(reactionNameLabel.snp.bottom).offset(verticalPadding)
+            make.leading.equalToSuperview().inset(20)
+            make.trailing.equalToSuperview().inset(20)
+        }
```

Comment:
you can add a bottom anchor. smtg like 
```
make.bottom.equalToSuperview().inset(20)
```
once you do this, your cell height will dynamically change based on much content you have on your label. It basically changes the height in order to also satisfy the bottom constraint. 
Once you do this, you can remove the function heightForRowAt in table view setup